security/nodeagent/templates/daemonset.yaml

```diff
@@ -1,47 +1,52 @@
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
-  name: istio-nodeagent
+  name: istio-sdsagent
   namespace: {{ .Release.Namespace }}
   labels:
-    app: istio-nodeagent
-    istio: nodeagent
+    app: istio-sdsagent
+    istio: sdsagent
     release: {{ .Release.Name }}
 spec:
   selector:
     matchLabels:
-      istio: nodeagent
+      istio: sdsagent
   template:
     metadata:
       labels:
-        app: istio-nodeagent
-        istio: nodeagent
+        app: istio-sdsagent
+        istio: sdsagent
         release: {{ .Release.Name }}
       annotations:
         sidecar.istio.io/inject: "false"
     spec:
-      serviceAccountName: istio-nodeagent-service-account
+      serviceAccountName: istio-sdsagent
 {{- if .Values.global.priorityClassName }}
       priorityClassName: "{{ .Values.global.priorityClassName }}"
 {{- end }}
       containers:
-        - name: nodeagent
-{{- if contains "/" .Values.nodeagent.image }}
-          image: "{{ .Values.nodeagent.image }}"
+        - name: sdsagent
+{{- if contains "/" .Values.sdsagent.image }}
+          image: "{{ .Values.sdsagent.image }}"
 {{- else }}
-          image: "{{ .Values.global.hub }}/{{ .Values.nodeagent.image }}:{{ .Values.global.tag }}"
+          image: "{{ .Values.global.hub }}/{{ .Values.sdsagent.image }}:{{ .Values.global.tag }}"
 {{- end }}
           imagePullPolicy: {{ .Values.global.imagePullPolicy | default "Always" }}
           args:
           {{- if .Values.global.logAsJson }}
             - --log_as_json
           {{- end }}
+            - --log_output_level=all:debug
```

Comment:
probably don't want this on by default?

Reply:
Right - I'm still testing. Will remove soon - but need to add some setting.